### ISSUE TITLE
Changes to README file

### ISSUE THREAD

**Issue by justin0022** [tl-its-umich-edu/my-learning-analytics#604]
The existing README file provides an overview for setting up the project, but doesn't really show its features. 

Perhaps we can move the existing README file to the `wiki` or some other setup file, and use the README to explain what it is we're trying to accomplish, with some screenshots and have a link to the [documentation](https://sites.google.com/umich.edu/my-learning-analytics-help/home) and a live demo. 

Thoughts?

**Comment by jonespm** [comment]
Yeah redoing all of the documentation as you mention would be something that would be really nice to do just often hard to get time for. The existing docs might be out of date and are only updated whenever someone new comes along and lets us know about a problem.

**Comment by justin0022** [comment]
Okay I'll try to draft something up

**Comment by justin0022** [comment]
Todo: add note about running the front-end in production mode.

**Comment by jennlove-um** [comment]
I'm closing this issue.  This is addressed in #661
